CS2113 IP/src/main/java/Duke.java

```diff
@@ -0,0 +1,108 @@
+import java.util.Scanner;
+
+public class Duke {
+    public static void Greet() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println(logo);
+        String horizontalLine = "________________________";
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        System.out.println(horizontalLine);
+    }
+
+    public static void listOperations() {
```

Comment:
maybe you could consider making this method slightly shorter? maybe in this case, you could directly compare whether the string is "bye", "list" etc instead of creating a boolean variable for all of them?